web/scripts/home/homeComponent.js

```diff
@@ -45,8 +47,9 @@ module.exports = {
       }
     })
 
-    const setProducts = (searchResults) => {
-      state = searchResults
+    const setProducts = (customCategories) => {
+      state.customCategories = customCategories
+      console.log(state.customCategories)
```

Comment:
Please remove the console.log() statement.

Reply:
👍